### PR TITLE
Fix pip in the centos8 image for building enterprise packages

### PR DESCRIPTION
**Pull request by m4dcoder**
Use the default pip3 that comes with the python3 install instead of overwriting the pip install.

### CODE REVIEW

File: buildpack/centos8/Dockerfile [hunk 1]
```diff
@@ -52,9 +52,7 @@ RUN yum -y install \
 
 # Python and tools
 RUN yum -y install python3 python3-devel python3-wheel && \
-    wget -qO - https://bootstrap.pypa.io/get-pip.py | python3 && \
-    pip install --upgrade "pip==19.1.1" && \
-    pip install setuptools virtualenv && \
+    pip3 install virtualenv && \
```
Comment (armab) [inline buildpack/centos8/Dockerfile:55]: Per https://github.com/StackStorm/st2packaging-dockerfiles/blob/78ed652b4a4c575a36b3ce2ec4be2ee93772cfcb/packagingbuild/centos8/Dockerfile#L90-L92
in community build dockerfiles

Can we also pin pip and virtualenv to have the same consistent environment across the builds?

Comment (m4dcoder) [reply buildpack/centos8/Dockerfile]: `pip==19.1.1` causes make rpm build error so I will not include that.  `rpmdevtools` is already up on the build tools list in the same Dockerfile.

File: buildpack/centos8/Dockerfile [hunk 2]
```diff
@@ -51,10 +51,9 @@ RUN yum -y install \
 
 
 # Python and tools
-RUN yum -y install python3 python3-devel python3-wheel && \
-    wget -qO - https://bootstrap.pypa.io/get-pip.py | python3 && \
-    pip install --upgrade "pip==19.1.1" && \
-    pip install setuptools virtualenv && \
+RUN yum -y install python3 python3-devel python3-wheel python3-virtualenv && \
+    pip3 install --user --upgrade pip==20.0.2 && \
```
Comment (armab) [inline buildpack/centos8/Dockerfile:55]:
```suggestion
    pip3 install --user --upgrade pip==19.1.1 && \
```
to keep the same version across all builds.

Comment (armab) [reply buildpack/centos8/Dockerfile:55]: And yes, indeed with the `--user` flag it works well.

After research it looks like related to https://github.com/pypa/pip/issues/5373 when the default package-distributed pip does something strange that prevents upgrading pip via pip.

**Comment by armab** [comment]
Merging to get the new Docker images deployed ASAP.

**Comment by armab** [comment]
https://github.com/pypa/pip/issues/5599 was the actual issue with pip.

**Pull request merged**